### PR TITLE
More Code Cleanup - When

### PR DESCRIPTION
**Pull request by walkerselby**
Merged chained if to when
Merge nested if statements
Add private fun calculateXPHour to CombatUtils to replace duplicate code in logic

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/FFGuideGUI.kt
Comment (hannibal002) [inline src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/FFGuideGUI.kt:1]: I'm not sure adding 10 lines just to remove the "else if" keywords is worth tbh

Comment (walkerselby) [reply src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/FFGuideGUI.kt:1]: Can revert this one if you want. But for this many chained, a when or switch is technically more efficient. Either way, I think it is slightly more readable. 

Separate from this, those hardcoded values feel abhorrent (though I am sure necessary)

Comment (CalMWolfs) [reply src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/FFGuideGUI.kt:1]: > Can revert this one if you want. But for this many chained, a when or switch is technically more efficient. Either way, I think it is slightly more readable.
> 
> Separate from this, those hardcoded values feel abhorrent (though I am sure necessary)

Agreed I didn't make them pretty but to remove them would require a lot of refactoring of the code

Comment (walkerselby) [reply src/main/java/at/hannibal2/skyhanni/features/garden/fortuneguide/FFGuideGUI.kt:1]: @hannibal002 let me know if you would like me to revert the changes in FFGuideGUI

**Review by hannibal002 — APPROVED** [review]

**Pull request merged**